core/tests/protractor.conf.js

```diff
@@ -156,9 +102,85 @@ exports.config = {
     ],
 
     users: [
-      'protractor_desktop/userJourneys.js',
+      'protractor_desktop/UserJourneys.js',
```

Comment:
Made this changes to test the behavior on the CI, once CI will fail I'll undo these changes!

Reply:
Tests are failing:
![image](https://user-images.githubusercontent.com/16653571/57574612-e4243400-7459-11e9-8d8b-065ee929a857.png)

![image](https://user-images.githubusercontent.com/16653571/57574614-ec7c6f00-7459-11e9-8fbb-2b3af9fc8900.png)